src/components/applications/ApplicationsView.js

```diff
@@ -29,7 +29,8 @@ export const ApplicationsView = ({getApplications, isHR, isDev, mailInBase64}) =
         setFetching(true)
         getApplications(id, mailInBase64, devPassword)
             .then(data => {
-                setApplications(data)
+                const applications = isDev ? data.filter(application => application.application.status !== "REJECTED") : data
```

Comment:
Dlaczego pozbywamy się rejected?

Reply:
bo nie ma sensu, żeby dev widział rejected aplikacje. On ma tylko przypisać zadanie do taska, a tak to dostaje wszystkie możliwe aplikacje. W ogóle nie wiem, czemu tu się pojawiały rejected XDDD

Reply:
Jest applicationStatus w Enums.js, można użyć i przy okazji przepisać na wielkie litery